src/Api/AdminConsole/Models/Response/Providers/ProviderOrganizationResponseModel.cs

```diff
@@ -54,6 +56,8 @@ public class ProviderOrganizationResponseModel : ResponseModel
     public DateTime RevisionDate { get; set; }
     public int UserCount { get; set; }
     public int? Seats { get; set; }
+    public int? AssignedSeats { get; set; }
```

Comment:
This is called `OccupiedSeats` elsewhere in the code - I suggest we keep the naming consistent.

Reply:
Resolved here: https://github.com/bitwarden/server/pull/4446/commits/c9ebe060db48d1fb3e0e310c4c5d24066389084b